custom-elements/custom-elements-registry/define.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<title>Custom Elements: Element definition</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#element-definition">
+<meta name="author" title="Koji Ishii" href="mailto:kojiishi@gmail.com">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<div id="log"></div>
```

Comment:
These metadata are from [templates](http://testthewebforward.org/docs/test-templates.html), required for csswg-test, optional for web-platform-tests. Checked a few tests in wpt, it looks like it's random. I'm fine either way, removed and added the spec link in the comment.